analysis/analyze_results_directory.py

```diff
@@ -1,81 +1,88 @@
 """Script to analyze experiments resulting from running the script
 analysis/run_supercloud_experiments.sh."""
 
-from typing import Tuple
+from typing import Tuple, Sequence
 import glob
 import dill as pkl
 import numpy as np
 import pandas as pd
 from predicators.src.settings import CFG
 
+GROUPS = [
+    # "ENV",
+    # "APPROACH",
+    # "EXCLUDED_PREDICATES",
+    "EXPERIMENT_ID",
+    # "NUM_TRAIN_TASKS",
+    # "CYCLE"
+]
 
-def create_dataframes() -> Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame]:
-    """Returns means, standard deviations, and sizes."""
+COLUMN_NAMES_AND_KEYS = [
```

Comment:
this is just for generating the pandas table right? can we leave most of this out and only keep what we care about (experiment_id, seed, num_solved, avg_num_preds, avg_test_time, avg_nodes_created, learning_time)